src/components/EnvelopeList.vue

```diff
@@ -269,7 +268,7 @@ export default {
 
 .list-enter-active,
 .list-leave-active {
-	transition: all 1s;
+	transition: all 100ms;
```

Comment:
```suggestion
	transition: all var(--animation-quick);
```

As per the variables.scss :)

Reply:
Thats what @skjnldsv would have said :)

Reply:
:see_no_evil: